packages/runtime-dom/src/devtools.ts

```diff
@@ -0,0 +1,6 @@
+import { setDevtoolsHook } from '@vue/runtime-core'
+
+const target: any = __BROWSER__ ? window : global
+
+target.__VUE__ = true
```

Comment:
this is a `side-effect`, it won't be included in the `esm-bundler` build file 🤔

Reply:
🤷